my-app/src/DATA/BrandsData.jsx

```diff
@@ -0,0 +1,216 @@
+import {v4 as uuid} from "uuid";
+import React from "react";
+export const productCardData = [
+  {
+    id: uuid(),
+    name: "V-Ribbed Belt",
+    category: "AIR_CONDITIONING",
+    image: {
+      src:
+        "https://boodmo.com/media/cache/catalog_part/images/parts/bb654de9e471092e85c66920aa5efb91.webp",
+      alt: "Rubber Belt"
+    },
```

Comment:
Question: Why did you make this DATA file outside of the backend folder? Edit the database in the db folder na.
Also, don't use id, use _id because the backend is made this way. For more information, check this discord link: https://discord.com/channels/914867137863118919/914867138148335657/954698270104494130

Reply:
Thanks for reviewing Partha .
Yeah! you are right but it's just for now . I will do it with backend only